### ISSUE TITLE
 robotframework-lsp does not recognize system directory separator ${/}

### ISSUE THREAD

**Issue by Miamaija** [robocorp/robotframework-lsp#153]
**Describe the bug**
I noticed that autocomplete doesn't work because robotframework-lsp does not recognize this system directory separator **${/}**.
I import resources from other directiories and I added those directories to the Python path. 

**To Reproduce**

VSCode settings.json:
```
"robot.pythonpath": [
      "C:\XXXX\XXX\XXX\",
    ],
```

Robot Framework file:
```
*** Settings ***
Resource          ..${/}XXXXXXX{/}XXXXXXXX${/}xxxxxxx_general.robot
Library           ..${/}..${/}ABC${/}EFG${/}XYZLibrary.py

```
**Expected behavior**

**Screenshots**

**Versions:**
 - OS: Windows 10
 - Robot Framework Version 3.2.2
 - Robot Framework Language Server Version 0.4.0
 - Client Version  VSCode 1.49.1

**Logs**

Here are a few lines from robotframework-lsp log:

```
lsp: 2020-09-23 21:03:04 UTC pid: 1808 - ThreadPoolExecutor-1_0 - INFO - robotframework_ls.impl.completion_context
Resource not found: C:\XXXX\XXX\XXX\..${/}XXXXXXX{/}XXXXXXXX${/}xxxxxxx_general.robot

lsp: 2020-09-23 21:03:05 UTC pid: 1808 - ThreadPoolExecutor-1_0 - DEBUG - robotframework_ls.impl.libspec_manager
Unable to find library named: ..${/}..${/}ABC${/}EFG${/}XYZLibrary.py
```

**Comment by fabioz** [comment]
I agree it should recognize it, but I'll just note that that's a pretty strange use of the separator (as far as I know, you should be able to just use `/` there without any issues as you could just write `Resource          ../../XXX/XX/resource_XXX_general.robot`).

That should work in Windows and Linux...

**Comment by Miamaija** [comment]
Thank you for the comment. I will change that to my files.
